.github/workflows/cron-nft-ttr.yml

```diff
@@ -0,0 +1,36 @@
+name: Measure NFT Time to Retrievability (nft-ttr)
+
+on:
+  schedule:
+    - cron: '30 * * * *'
+  workflow_dispatch:
+
+jobs:
+  measure:
+    name: measure nft time to retrievability
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        env: ['staging', 'production']
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - uses: bahmutov/npm-install@v1
+      - name: Run job
+        env:
+          DEBUG: '*'
+          ENV: ${{ matrix.env }}
+          NFT_STORAGE_API_KEY: ${{ secrets.NFT_STORAGE_API_KEY }}
+          PUSHGATEWAY_BASIC_AUTH: ${{ secrets.PUSHGATEWAY_BASIC_AUTH }}
```

Comment:
I haven't been able to test the pathway when this is truthy because I don't have a sample secret

Reply:
nvm I found a value in secrets vault and tested locally. then made small fixes until the push metrics request responded with status 200